Makefile

```diff
@@ -28,6 +28,10 @@ endif
 vendor: ## Run composer install
 	$(COMPOSER_RUN) install $(COMPOSER_INSTALL_PARAMETERS)
 
+.PHONY: ibexa-assets
+ibexa-assets: ## Generate Ibexa Admin UI assets
+	$(COMPOSER_RUN) ibexa-assets
```

Comment:
I do not think this would work without calling nvm? In the end, `ibexa-assets` is running `yarn ibexa` and that means we need to have correct node version for this.

Reply:
Nvm statements added to ibexa-assets command